SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -415,9 +415,11 @@ Type 4 conversion equations are intended for data produced by Simrad EK80 and ES
 
 == Coordinate systems
 
-This section provides a complete mathematical overview of the coordinate systems necessary to physically locate the position of an echo relative to a fixed global coordinate system when measured from a moving or stationary platform. The coordinate systems detailed here follow those used in some multibeam bathymetric mapping sonars, but in many cases this level of preciseness will be unnecessary and several coordinate systems will overlap.
+This section provides a complete mathematical overview of the coordinate systems necessary to physically locate the position of an echo relative to a geographic coordinate system when measured from a moving or stationary platform. The coordinate systems detailed here follow those used in some multibeam bathymetric mapping sonars, but in many cases this level of preciseness will be unnecessary and several coordinate systems will overlap.
 
-There are five right-handed Cartesian coordinate systems (<<coordinateSystemList>>) and one Cartesian angular coordinate system used to specify the physical location of received echoes. Some of the coordinate systems have different origins and this is implemented as a translation vector in the input coordinate system. Rotation angles are as per the right-hand rule.
+There are four right-handed Cartesian coordinate systems (<<coordinateSystemList>>, <<coordinate_system_figure>>) associated with the platform, transducer(s), and acoustic beams, and one Cartesian angular coordinate system used to specify the physical location of received echoes with a sonar beam. Some of the coordinate systems have different origins and this is implemented as a translation vector in the input coordinate system. Rotation angles are as per the right-hand rule.
```

Comment:
There are four right-handed Cartesian coordinate systems (<<coordinateSystemList>>, <<coordinate_system_figure>>) associated with the platform, transducer(s), and acoustic beams, used to specify the physical location of received echoes with a sonar beam. Some of the coordinate systems have different origins and this is implemented as a translation vector in the input coordinate system. Rotation angles are as per the right-hand rule.
Comment: the position of the echoes in the Sonar beam coordinate system is obtained with the range and the split aperture angles and no extra angular coordinate system needs to be defined

Reply:
I was considering the split aperture angles as another coordinate system, but you are right that they are just a different way to specify a point in the beam coordinate system.